POS/Operation/02_SelectOffer.md

```diff
@@ -87,34 +155,116 @@ Offers for POS are created, see [Manage offers for POS](../Integration/07_Manage
     > [Info] The search already provides you with proposals for your number as you type.
 
 
-3. Select the desired offer by clicking the corresponding product in the list.   
-    The offer is added to the bill list on the right side.
+3. Select the desired offer by clicking the corresponding offer in the list.   
+    The offer is added to the bill list if no further features such as an [age verification](#verify-the-age) or a [product option](#add-a-product-option) are interposed. 
 
-    > [Info] The number remains in the search bar when you add a product to the bill list. When you click the search bar with that number again, the search results list for this number is displayed again.
+    > [Info] The number remains in the search bar when you add an offer to the bill list. When you click the search bar with that number again, the search results list for this number is displayed again.
 
 
+### Select a pre-printed voucher offer 
 
-## Use single scan
+Select a voucher offer that allows you to identify and charge a pre-printed voucher. You can then scan the voucher code with a barcode scanner or add the card code manually. 
+
+#### Prerequisites
+
+- Your company uses the KNISTR integration, with which vouchers can be pre-printed.
+
+#### Procedure
+
+*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 
+
+![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")
+
+1. Click the *Find offer by nam...* search bar in the upper left part of the tab.  
+    The cursor is displayed in the search bar.
+
+2. Enter the name or the SKU of the voucher you are searching for in the search bar.
+    The offers that match the search are displayed in a list below the search bar.
+
+    > [Info] The search already provides you with proposals as you type.
+
+3. Select the desired voucher by clicking the corresponding offer in the list.   
```

Comment:
Select the desired voucher offer by clicking the desired one in the list?

(Pre-printed) voucher offer -> pre-printed würde ich vielleicht weglassen (wenn, dann nur das erste Mal und dann erklären). Voucher offer im Sinne von "voucher" als Produkt zu kaufen wird manchmal nicht so deutlich. Eigentlich es ist einfach ein anderes Produkt, das man kaufen kann, und wo man noch Geldwert darauf aufladen kann, oder?